webapp/src/components/bot_slector.tsx

```diff
@@ -27,7 +28,10 @@ export const DropdownBotSelector = (props: DropdownBotSelectorProps) => {
         >
             <>
                 <SelectMessage>
-                    {'Generate With:'}
+                    <FormattedMessage
+                        id='dropdown_bot_selector.generate_with'
```

Comment:
Do we need the id field? We didn't use it on Playbooks and never ran into any issues.

Reply:
I think is not strictly needed, I can generate the sha of the text as IDs on the fliy, but it is consistent with mattermost-webapp. The only other drawback is the accidental collation of translations, sometimes, same text in different places have different translations.

Reply:
I did the changes because I think it is also less collision prone with other services, and even to ensure that more, I'm using 8 characters ids, instead of 6, to avoid any accidental collition on translations between playbooks and copilot. Could be that something have the same text but totally different context, this would collide ever.